samples/business-process-samples/product/sample-client/src/main/java/org/wso2/ei/businessprocess/samples/client/SamplesInvoker.java

```diff
@@ -98,11 +99,15 @@ private static void loadServiceProperties(String propFilePath, String operation)
             System.exit(0);
         } finally {
             try {
-                fin.close();
+                if (fin != null) {
+                    fin.close();
+                }
             } catch (IOException e) {

```

Comment:
Since we are going close the stream, we don't care if errors occurred. This is the existing impl, I also think it's correct.